iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/execution/load/LoadTsFileManager.java

```diff
@@ -67,6 +71,7 @@ public class LoadTsFileManager {
   private static final String MESSAGE_WRITER_MANAGER_HAS_BEEN_CLOSED =
       "%s TsFileWriterManager has been closed.";
   private static final String MESSAGE_DELETE_FAIL = "failed to delete {}.";
+  private static final String METRIC_POINT_IN = "pointsIn";
```

Comment:
There are `METRIC_POINT_IN` in AbstractMemtable. So maybe it's a good idea to extract this parameter into const.